salt/orch/update.sls

```diff
@@ -29,10 +29,15 @@
 {%- set is_updateable_master_tgt = is_updateable_tgt + ' and ' + is_master_tgt %}
 {%- set is_updateable_worker_tgt = is_updateable_tgt + ' and ' + is_worker_tgt %}
 {%- set is_updateable_node_tgt   = '( ' + is_updateable_master_tgt + ' ) or ( ' + is_updateable_worker_tgt + ' )' %}
-
 {%- set all_masters = salt.saltutil.runner('mine.get', tgt=is_master_tgt, fun='network.interfaces', tgt_type='compound').keys() %}
 {%- set super_master = all_masters|first %}
 
+{%- set is_migration = salt['pillar.get']('migration', false) %}
+
+{% if is_migration %}
+{%- set migrated_nodes = is_responsive_node_tgt + 'and G@migration_in_progress:true' %}
```

Comment:
i think there is a whitespace missing? ' and G@migration_in_progress:true'

error from salt master: `Invalid compound target: *and G@migration_in_progress:true`